src/main/java/org/broadinstitute/consent/http/db/mapper/DataAccessRequestMapper.java

```diff
@@ -21,6 +21,7 @@ public DataAccessRequest map(ResultSet resultSet, StatementContext statementCont
         DataAccessRequest dar = new DataAccessRequest();
         dar.setId(resultSet.getInt("id"));
         dar.setReferenceId(resultSet.getString("reference_id"));
+        dar.setCollectionId(resultSet.getInt("collection_id"));
```

Comment:
This could be written more safely if we missed a query. Look at the `hasColumn` method where we could check if the query has the field, then we would set the value.